setup.cfg

```diff
@@ -79,7 +80,6 @@ setenv =
     PIP_DISABLE_PIP_VERSION_CHECK = 1
     COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
     {py27,pypy}: PYTHONWARNINGS=ignore:DEPRECATION::pip._internal.cli.base_command
-    DBT_PROFILES_DIR = {env:DBT_PROFILES_DIR:{toxinidir}/tests/dbt_project}
```

Comment:
Could you keep one of the tox env with `DBT_PROFILES_DIR` to test for both scenario's?

Reply:
done https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/a5420f3f1feac5d753d2cae1cd002bfa51e8862f